app/org/sagebionetworks/bridge/models/SynapseAccount.java

```diff
@@ -0,0 +1,49 @@
+package org.sagebionetworks.bridge.models;
+
+/**
+ * Represent a Synapse Account
+ */
+public class SynapseAccount {
```

Comment:
Why is this file part of this pull request? (Similarly SynapseUserIdHolder, SynapseAccountTest, and SynapseUserIdHolderTest.)